src/components/Event/EventItem.tsx

```diff
@@ -0,0 +1,118 @@
+import { Event, EventParticipants, EventStatus, User } from "@prisma/client";
+import { EventList } from "./EventList";
+import Image from "next/image";
+import { CiCalendarDate, CiBookmarkCheck } from "react-icons/ci";
+import { HiOutlineTicket } from "react-icons/hi2";
+import { MdOutlinePublic } from "react-icons/md";
+import { RiGhostSmileLine } from "react-icons/ri";
+import { MouseEventHandler } from "react";
+import { EventItemStatus } from "./EventItemStatus";
+
+export interface iEventItem {
```

Comment:
I believe in TypeScript we don't identify interfaces with this naming convention, especially when the interface is being used to describe props for a component. I believe it should be named `EventItemProps`, but you don't have to change it right now. We might/likely would refactor this and we can rename it then

Reply:
Got it!